README.md

```diff
@@ -223,7 +259,27 @@ ansible-test sanity
 The [molecule/README.md](molecule/README.md) file has instructions on testing the deployment on a virtual machine (VM).
 By default, the VM provider is [testing-farm.io](https://docs.testing-farm.io/).
 
+## Contributing
+
+## Support
+
+Support tickets for RedHat Trusted Artifact Signer can be opened at https://access.redhat.com/support/cases/#/case/new?product=Red%20Hat%20Trusted%20Artifact%20Signer.
+
+## Release notes and Roadmap
+
+Release notes can be found [here](https://docs.redhat.com/en/documentation/red_hat_trusted_artifact_signer/1/html/release_notes/index).
```

Comment:
This is the wrong link to the official release notes for RHTAS 1.1.  The new link will be --> https://docs.redhat.com/en/documentation/red_hat_trusted_artifact_signer/1.1/html-single/release_notes/index 

This link currently gives you a 404 page error because the RHTAS 1.1 Release Notes have not been published yet, but this link will work once the release notes get published on release day.

Reply:
Awesome thanks